spec.html

```diff
@@ -7776,6 +7776,101 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
+                Any, default value is *undefined*.
+              </td>
+              <td>
+                Field reserved for use by hosts.
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-hostmakejobcallback" aoid="HostMakeJobCallback">
+      <h1>HostMakeJobCallback ( _callable_ )</h1>
+      <p>HostMakeJobCallback is a host-defined abstract operation that takes argument _callable_ (a callable object).</p>
+      <p>The implementation of HostMakeJobCallback must conform to the following requirements:</p>
+      <ul>
+        <li>It must always complete normally (i.e., not return an abrupt completion).</li>
+        <li>It must always return a JobCallback Record whose [[Callback]] field is _callable_.</li>
+      </ul>
+      <p>The default implementation of HostMakeJobCallback performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: IsCallable(_callable_) is *true*.
+        1. Return the JobCallback Record { [[Callback]]: _callable_, [[HostDefined]]: ~empty~ }.
+      </emu-alg>
+      <p>ECMAScript hosts that are not web browsers must use the default implementation of HostMakeJobCallback.</p>
+      <emu-note>
+        <p>This is called at the time that the callback is passed to the function that is responsible for its being eventually scheduled and run. For example, `promise.then(thenAction)` calls MakeJobCallback on `thenAction` at the time of invoking `Promise.prototype.then`, not at the time of scheduling the reaction Job.</p>
+      </emu-note>
+    </emu-clause>
+
+    <emu-clause id="sec-hostcalljobcallback" aoid="HostCallJobCallback">
+      <h1>HostCallJobCallback ( _jobCallback_, _V_, _argumentsList_ )</h1>
+      <p>HostCallJobCallback is a host-defined abstract operation that takes arguments _jobCallback_ (a JobCallback Record), _V_ (an ECMAScript language value), and _argumentsList_ (a List of ECMAScript language values).</p>
+      <p>The implementation of HostCallJobCallback must conform to the following requirements:</p>
+      <ul>
+        <li>It must always perform and return the result of Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).</li>
+      </ul>
+      <emu-note>
+        <p>This requirement means that hosts cannot change the [[Call]] behaviour of callable objects defined in this specification.</p>
+      </emu-note>
+      <p>The default implementation of HostCallJobCallback performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: IsCallable(_jobCallback_.[[Callback]]) is *true*.
+        1. Return ? Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).
+      </emu-alg>
+      <p>ECMAScript hosts that are not web browsers must use the default implementation of HostMakeJobCallback.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-calljobcallback" aoid="CallJobCallback">
+      <h1>CallJobCallback ( _jobCallback_, _V_ [ , _argumentsList_ ] )</h1>
```

Comment:
Both of this AO's callsites provide the `_argumentsList_` parameter. I don't think this AO needs to exist at all; `HostCallJobCallback` can just require the `_argumentsList_` parameter.

Reply:
This was introduced as a convenience wrapper to align with the `Call` AO, which also has an optional _argumentsList_. Happy to remove for now, but might reintroduce if it ends up getting used incorrectly in the future.